src/test/java/net/openhft/chronicle/queue/AcquireReleaseTest.java

```diff
@@ -69,7 +69,7 @@ public void onReleased(int cycle, File file) {
                 .build()) {
 
             int iter = 4;
-            try (ExcerptAppender excerptAppender = queue.acquireAppender()) {
+            try (ExcerptAppender excerptAppender = queue.createAppender()) {
```

Comment:
All the instances that look like this are a find and replace, there are some cases where it's less simple